src/levels/level_1.gd

```diff
@@ -8,6 +8,8 @@ func _ready() -> void:
 	super._ready()
 	$Terminal.interacted.connect(show_puzzle)
 	$PuzzleConsole.puzzle_complete.connect(puzzle_completed)
+	$LevelTransition.open()
+	$EndTrigger.body_entered.connect(level_trigger_entered)
```

Comment:
Along with my other comment, the LevelTransition shouldn't need to be manually started in every single level. 

Right now, the base level is just a script and not a whole scene, meaning we can't just put the level transition in a scene there. Nonetheless, we can edit `base_level.gd` to spawn in the a level transition, add it as a child, and start the animation, so it will automatically happen in every level

Reply:
Ooh good point.